analysis/post_release/manuscript_tableAER.R

```diff
@@ -0,0 +1,57 @@
+# Specify parameters -----------------------------------------------------------
+print('Specify parameters')
+
+perpeople <- 100000 # per X people
+
+# Load data --------------------------------------------------------------------
+print('Load data')
+
+df <- read.csv("output/post_release/lifetables_compiled.csv")
+
+# Filter data ------------------------------------------------------------------
+print("Filter data")
+
+df <- df[df$aer_age=="overall" &
+           df$aer_sex=="overall" &
+           df$analysis=="main" & 
+           df$days==196,]
+
+# Add plot labels --------------------------------------------------------------
+print("Add plot labels")
+
+plot_labels <- readr::read_csv("lib/plot_labels.csv",
+                               show_col_types = FALSE)
```

Comment:
I can't see the file in lib, is it there ?

Reply:
No, I just added the scripts.

Reply:
Lib file added.